app/src/main/java/cookpad/com/bottomnavwatson/DetailWithoutBottomMenuFragment.kt

```diff
@@ -0,0 +1,62 @@
+package cookpad.com.bottomnavwatson
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavDeepLinkBuilder
+import androidx.navigation.fragment.NavHostFragment.findNavController
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.setupWithNavController
+import kotlinx.android.synthetic.main.detail_fragment.toolbar
+import kotlinx.android.synthetic.main.detail_fragment.tvTestDeepLinks
+import java.util.Random
+
+class DetailWithoutBottomMenuFragment : Fragment(R.layout.detail_without_bottom_menu_fragment) {
```

Comment:
As `DetailWithoutBottomMenuFragment` is almost identical to `DetailFragment`, please use the `SafeArgs` to supply a parameter called `hideNavBar` or alike based on which the layout and the visibility of the navbar will be set.

Reply:
Done